### ISSUE TITLE
moveTo deprecated, use append instead

### ISSUE THREAD

**Issue by marcoacierno** [ben-eb/postcss-merge-rules#52]
It seems that cssnano generates this warning because at the line 192 (https://github.com/ben-eb/postcss-merge-rules/blob/master/src/index.js#L192) the deprecated method `moveTo` is used: `decl.moveTo(cache);`

**Comment by ben-eb** [comment]
We're not compatible with PostCSS 6 yet, it should be out soon.

**Comment by marcoacierno** [comment]
oh cool then, sorry :)

**Comment by ben-eb** [comment]
Resolved in https://github.com/ben-eb/cssnano/releases/tag/v4.0.0-rc.0.

**Comment by pldg** [comment]
I'm getting this error too, using version `2.1.2`.

My plugins list:

```javascript
plugins: {
  'postcss-import': {},
  'postcss-css-variables': {preserve: false},
  'postcss-color-function': {},
  'postcss-image-set-polyfill': {},
  'postcss-svgo': {},
  'postcss-flexbugs-fixes': {},
  'postcss-merge-rules': {},
  'postcss-discard-empty': {},
  'autoprefixer': {browsers: ['last 2 versions', 'IE >= 10']}
}
```

How can I fix it? 
Thanks!
